src/components/FormRegister.vue

```diff
@@ -203,6 +209,49 @@ export default defineComponent({
       <q-separator color="grey-2" class="q-my-lg" />
       <!-- Buttons: Register with 3rd party -->
       <login-register-buttons variant="register" />
+      <!-- Link: Register Coordinator -->
+      <div class="q-mt-xl">
+        <div
+          class="q-pa-md text-body2 text-white"
+          :style="{
+            'background-color': backgroundColor,
+            'border-radius': borderRadius,
+          }"
+          data-cy="form-register-coordinator"
+        >
+          <p
+            class="q-mt-none q-mb-md"
+            data-cy="form-register-coordinator-description"
+          >
+            {{ $t('register.form.hintRegisterAsCoordinator') }}
+          </p>
+          <p
+            class="q-mt-md q-mb-none"
+            data-cy="form-register-coordinator-link-wrapper"
+          >
+            <a
```

Comment:
Same as my comment bellow https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/121/files#r1390659229.

If route path /#/register-coordinator doesn't exists yet create some fake with existed layout/component.

Reply:
Added using the `router-link`. Also added router to component test.
I also fixed this issue in `FormLogin` as the use was identical.
Added route `register-coordinator` (for now, it has the same content as `register`)